### PR TITLE
Add Laravel 8 support

### PR DESCRIPTION
**Pull request by jfernancordova**
Issue #25

### CODE REVIEW

**Comment by Giacomo92** [comment]
Hi @jfernancordova , can fix also CI please?

Thank you for your contribution

**Comment by jfernancordova** [comment]
> Hi @jfernancordova , can fix also CI please?
> 
> Thank you for your contribution

You're welcome, of course, the upgrade guide (https://laravel.com/docs/8.x/upgrade#php-7.3.0-required) says laravel 8 is compatible from PHP 7.3, the previous version should be removed, so I will remove on the cicle CI builds the 7.1 and 7.2.

**Comment by jfernancordova** [comment]
@Giacomo92 hi!, what do you think? All checks passed, let me know if you want another fix, thx.

**Comment by Giacomo92** [comment]
Hi @jfernancordova , check comments in the code

**Comment by jfernancordova** [comment]
> Hi @jfernancordova , check comments in the code
 @Giacomo92 specifically what part of the code?, I can't see new comments

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 language: php
 
 php:
-  - 7.2
-  - 7.3
```
[the remark this answers is not in the document]
Comment (jfernancordova) [reply .travis.yml:5]: You're right, I'll fix it.

**Review by Giacomo92 — APPROVED** [review]

**Comment by Giacomo92** [comment]
Thank your for your PR @jfernancordova . Good job!

**Comment by jfernancordova** [comment]
> Thank your for your PR @jfernancordova . Good job!
You're welcome, thank you, very appreciated!

**Pull request merged**